packages/rgbpp/src/rgbpp/summary/asset-summarizer.ts

```diff
@@ -0,0 +1,98 @@
+import { Cell } from '@ckb-lumos/base';
+import { Utxo, encodeUtxoId } from '@rgbpp-sdk/btc';
+import { leToU128, encodeCellId } from '@rgbpp-sdk/ckb';
+
+export interface AssetSummary {
+  amount: bigint;
+  utxos: number;
```

Comment:
When is `AssetSummary.utxos` field used? Is it necessary?
Because each `AssetSummary` represents the assets bound to a single BTC UTXO,
when an `AssetSummary` is created, `utxoCount` is always be `1`, right?

Update:
ref: https://github.com/ckb-cell/rgbpp-sdk/blob/b1865c2ac35ef3dec68afbd81e0736cbff80edd0/packages/rgbpp/src/rgbpp/summary/asset-summarizer.ts#L81

Reply:
> When is AssetSummary.utxos field used? Is it necessary?

The `summary` object in the result is also for users to review or display on an app. So, even though some props are not used in our code, they can still help users identify which UTXOs/Cells are included/excluded during the process, and how many of them are included/excluded.

> Because each AssetSummary represents the assets bound to a single BTC UTXO, when an AssetSummary is created, utxoCount is always be 1, right?

Each `AssetSummary` represents the summary of a single RGBPP XUDT Asset instead of a single UTXO, therefore the `utxoCount` can be greater than `1`.

Reply:
I see.